src/test/java/com/example/finance/listener/BudgetEntityListenerTest.java

```diff
@@ -0,0 +1,51 @@
+package com.example.finance.listener;
+
+import com.example.finance.factory.BudgetMockFactory;
+import com.example.finance.model.entity.BudgetEntity;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest
+@ContextConfiguration(classes = {BudgetEntityListener.class})
+class BudgetEntityListenerTest {
```

Comment:
I think you don't need to test that class because it was only for our meeting purpose. If you don't have any idea to create your own listener - let me know. I will figure out something :)